app/assets/stylesheets/styles/game.scss

```diff
@@ -0,0 +1,53 @@
+/*// * {margin: 0; padding: 0; }
+// table { border-collapse: collapse; border-spacing: 0; }
+//
+// #chessboard { padding: 0px; margin: 0 auto; border: 2px solid #181818; }
+// #chessboard tr td { width: 60px; height: 60px; }
+// #chessboard tr:nth-child(2n) td:nth-child(2n+1) { background: #9f9f9f; }
+// #chessboard tr:nth-child(2n+1) td:nth-child(2n) { background: #9f9f9f; }*/
+
+#chessboard {
+  border-collapse: collapse;
+  margin: 0 auto .5em auto;
+  padding-top: 40px;
+  width: auto;
+  border: 3px solid black;
+}
+
+#chessboard tr:nth-child(even) td:nth-child(even),
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3